.github/workflows/PROJECT-COMMON-AUTO-CHANGELOG-CONTROL.yaml

```
@@ -40,218 +40,120 @@ jobs:
           if [ -f ".github/scripts/version_manager.sh" ]; then
             chmod +x .github/scripts/version_manager.sh
-            echo "✅ 버전 관리 스크립트 권한 설정 완료"
-          else
-            echo "⚠️ .github/scripts/version_manager.sh 파일을 찾을 수 없습니다."
+            echo "✅ 버전 스크립트 권한 설정 완료"
           fi
 
-      - name: CodeRabbit 제목 변경 요청
+      - name: 버전 정보 확인
+        id: get_version
         run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:29:28: Double quote to prevent globbing and word splitting

(shellcheck)